### PR TITLE
Add benchmark for Catboost modelbuilder

### PR DESCRIPTION
**Pull request by TroninDV**
Add benchmark and config for Catboost modelbuilder.

### CODE REVIEW

**Comment by avolkov-intel** [comment]
Scripts and config files look correct, inference metrics are the same for default prediction and prediction with model builders

**Comment by razdoburdin** [comment]
Hi @Alexsandruss , can we start a review process of this PR? All model builders related activities are moving forward fast now.

File: modelbuilders_bench/catboost_mb.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    metric_name = ['accuracy', 'log_loss']
+    metric_func = [bench.accuracy_score, bench.log_loss]
+    if 'cudf' in str(type(y_train)):
+        params.n_classes = y_train[y_train.columns[0]].nunique()
+    else:
+        params.n_classes = len(np.unique(y_train))
+
+    # Covtype has one class more than there is in train
+    if params.dataset_name == 'covtype':
+        params.n_classes += 1
```
Comment (Alexsandruss) [inline modelbuilders_bench/catboost_mb.py:120]: @razdoburdin, this should be aligned to https://github.com/IntelPython/scikit-learn_bench/pull/125

File: modelbuilders_bench/catboost_mb.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+    if params.n_classes > 2:
+        cb_params['bootstrap_type'] = 'Bernoulli'
+        cb_params['classes_count'] = params.n_classes
+        cb_params['objective'] = 'MultiClass'
+    else:
+        cb_params['scale_pos_weight'] = params.scale_pos_weight
+        cb_params['objective'] = 'Logloss'
+
+t_creat_train, dtrain = bench.measure_function_time(cb.Pool, X_train, params=params,
```
Comment (Alexsandruss) [inline modelbuilders_bench/catboost_mb.py:130]: `creat` is bad shorten name

File: configs/modelbuilders/catboost_mb_config.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+                    }
+                }
+            ],
+            "max-depth": 8,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 256
+        }
+    ]
+}
```
Comment (Alexsandruss) [inline configs/modelbuilders/catboost_mb_config.json:294]: Add line break at end of file

File: configs/modelbuilders/catboost_mb_config.json [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+                    "testing": {
+                        "x": "data/airline_x_test.npy",
+                        "y": "data/airline_y_test.npy"
+                    }
+                }
+            ],
+            "max-depth": 8,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 256
```
Comment (Alexsandruss) [inline configs/modelbuilders/catboost_mb_config.json:202]: These params are repeated too many times, I suggest to move it to `common`.

**Pull request merged**